coops-ui/src/views/AnnualReport.vue

```diff
@@ -363,23 +358,32 @@ export default {
 
     directorsChange (val) {
       // when directors change, update filing data
-      console.log('AnnualReport, directorsChange =', val)
+      // console.log('AnnualReport, directorsChange =', val)
```

Comment:
In the future can we just delete these? At some point the linter should be set up to complain about commented out code.

Reply:
Yes, I am deleting this (per Katie's review).